xrviz/fields.py

```diff
@@ -64,6 +99,48 @@ def change_y(self, value):
         values.remove(self.x.value)
         self.y.options = values
 
+    def create_agg_plot(self, *args):
+        remaining_dims = [dim for dim in self.var_dims if dim not in self.dims_selected_for_agg]
+        assign_opts = {dim: self.data[dim] for dim in remaining_dims}
+
+        # the method getattr(data.air_u, self.aggregation.value ,[*self.dims_selected_for_agg])
+        # does not works as expected
+        if self.aggregation.value == 'mean':
```

Comment:
Each statement has been written separately  because the method `getattr(data.air_u, self.aggregation.value ,[*self.dims_selected_for_agg])` does not works as expected. 
For example data: `getattr(data.air_u, 'mean' ,['time'])()` returns 
```
<xarray.DataArray 'air_u' ()>
array(1.03431, dtype=float32)
```
which is not what we want. Please let me know if there is correct way to do this. Else I will try raising an issue.

Reply:
```
getattr(data.air_u, 'mean')(['time'])
```

Reply:
I have changed according to this. Thanks!